internal/repository/verify_test.go

```diff
@@ -28,6 +28,10 @@ func TestVerifyRef(t *testing.T) {
 	entryID := common.CreateTestRSLReferenceEntryCommit(t, repo.r, entry, gpgKeyBytes)
 	entry.ID = entryID
 
+	if err := policy.Apply(context.Background(), repo.r, false); err != nil {
```

Comment:
shouldn't this happen in the createTestRepositoryWithPolicy helper?

Reply:
I see it there as well.

Reply:
Removed